experimental/traffic-portal/src/app/core/parameters/detail/parameter-detail.component.html

```diff
@@ -0,0 +1,51 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<mat-card>
+	<tp-loading *ngIf="!parameter"></tp-loading>
+	<form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="parameter">
+		<mat-card-content>
+			<mat-form-field *ngIf="!new">
+				<mat-label>ID</mat-label>
+				<input matInput type="text" name="id" disabled readonly [defaultValue]="parameter.id" />
+			</mat-form-field>
+			<mat-form-field>
+				<mat-label>Name</mat-label>
+				<input matInput type="text" name="name" required [(ngModel)]="parameter.name" />
+			</mat-form-field>
+			<mat-form-field>
+				<mat-label>Config File</mat-label>
+				<input matInput type="text" name="configFile" required [(ngModel)]="parameter.configFile" />
+			</mat-form-field>
+			<mat-form-field>
+				<mat-label>Value</mat-label>
+				<input matInput type="text" name="value" required [(ngModel)]="parameter.value" />
+			</mat-form-field>
+			<mat-form-field>
+				<mat-label>Secure</mat-label>
+				<mat-select name="secure"[(ngModel)]="parameter.secure" required>
+					<mat-option *ngFor="let s of secure" [value]="s.value">{{s.label}}</mat-option>
+				</mat-select>
+			</mat-form-field>
```

Comment:
for boolean properties, we should use a [checkbox](https://material.angular.io/components/checkbox/overview) or a [slide-toggle](https://material.angular.io/components/slide-toggle/overview) instead of a select.